frontend/docs/docs/user-guide/presentation-sharing.md

```diff
@@ -0,0 +1,24 @@
+# Collection Presentation & Sharing
+
+## Sharing Collections
+
+Collections are private by default, but can be made sharable either with an unlisted link, or displayed in your org's public gallery for everyone to discover. Sharing settings can be found within the **Sharing** tab of the Collection Settings dialog. Public collections will not be visible unless your org has enabled its [public collections gallery](org-settings.md#public-collections-gallery).
+
+After a collection has been made public the collection can also be embedded into other websites using the provided embed code. Un-sharing the collection will break any previously shared links and remove it from your gallery.
+
+For further resources on embedding archived web content into your own website, see the [ReplayWeb.page docs page on embedding](https://replayweb.page/docs/embedding).
+
+## Initial View
+
+Set the Initial View to control what visitors see first when viewing a collection. The initial view can either be a list of pages (ReplayWeb.page's default), or a single page from your collection such as a crawl start URL or index page.
+
+## Thumbnails
+
+The collection thumbnail image will be displayed as a card within your org's public gallery. This thumbnail can be edited from the **Presentation** tab of the Collection Settings dialog. Thumbnails can either be a screenshot of a page from the collection, or a placeholder provided by Webrecorder.
+
+??? question "Why don't the pages in my collection have thumbnails?"
```

Comment:
IMO this is more direct language as to how a missing thumbnail would manifest in the UI ("show"):
```suggestion
??? question "Why isn't there a thumbnail preview for my page?"
```